### PR TITLE
feat: add helm chart support for multiple expanders including "priority"

### PR DESCRIPTION
**Pull request by JoshVee**
#### Which component this PR applies to?

helm charts

#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Add support for multiple expanders to the Helm chart, as per https://github.com/kubernetes/autoscaler/pull/4233.

This change allows the "priority" expander to be used along with other expanders.

#### Which issue(s) this PR fixes:

N\A

#### Does this PR introduce a user-facing change?

```release-note
Add Helm support for multiple expanders including "priority".
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: charts/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.13.1
+version: 9.14.0
```
Comment (JoshVee) [inline charts/cluster-autoscaler/Chart.yaml:20]:
```suggestion
version: 9.14.0

```

**Comment by gjtempleton** [comment]
Thanks for this!
/lgtm
/approve

**Comment by AndrewFarley** [comment]
@gjtempleton Please review this MR against the other one I did, I don't have time to confirm, but based on a brief look at the changelogs, you may have missed a few places to ensure this MR does what it should.  See: #4503

**Pull request merged**